src/PKSim.CLI.Core/Services/SimulationExporter.cs

```diff
@@ -127,13 +127,17 @@ public Task Export(Simulation simulation, SimulationExportOptions simulationExpo
          var exportMode = simulationExportOptions.ExportMode;
 
          if (exportMode.HasFlag(SimulationExportMode.Csv) && individualSimulation != null)
-            tasks.Add(exportIndividualSimulationToCsvAsync(individualSimulation, simulationExportOptions));
+            tasks.Add(exportSimulationResultsToCsv(individualSimulation, simulationExportOptions));
+
+         // Population always exported no matter what
+         if(populationSimulation != null)
+            tasks.Add(exportPopulationToCsvAsync(populationSimulation, simulationExportOptions));
```

Comment:
@Yuri05 This is new here. Always exporting

Reply:
this will also export the aging data, right?

Reply:
when exporting a pop simulation with aging for cluster: aging parameters are exported...
Can you add it here?

Reply:
yep (never exported as part as the qualification export I meant)

Reply:
done. Also removing the append method for log that is not supported by serilog anymore